packages/dx-react-grid-bootstrap3/package.json

```diff
@@ -32,6 +32,7 @@
   "files": [
     "dist"
   ],
+  "types": "dist/index.d.ts",
```

Comment:
Can we rename `index.d.ts` to `dx-react-grid-bootstrap3.d.ts`? It will be identically to other files in the `dist` folder.